lib/ui/window.dart

```diff
@@ -72,6 +72,40 @@ enum AppLifecycleState {
   suspending,
 }
 
+/// A representation of the insets relative to each side of the window into
+/// which the application can render, but over white the operating system will
+/// likely place system UI, such as the keyboard, that fully obscures any
+/// content. These insets are exposed by [Window.viewInsets].
+///
+/// For a generic class that represents distances around a rectangle, see the
+/// [EdgeInsets] class.
+///
+/// See also:
+///
+///  * [WidgetsBindingObserver], for a widgets layer mechanism to receive
+///    notifications when the margin changes.
+///  * [MediaQuery.of], a simpler mechanism for the same.
```

Comment:
I feel this is not super clear. Is it referring to 'the same' representation of the insets or of the mechanism to receive notifications or something else?

Reply:
Updated both here and for `padding` for consistency.